modules/ROOT/pages/sap/sap-reference-45.adoc

```diff
@@ -0,0 +1,815 @@
+
+
+== Configurations
+---
+[[inbound-config]]
+=== Inbound
+
+Configure the default configuration for SAP extension inbound elements.
+
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+|Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
+| Connection a| <<inbound-config_simple-connection-provider, Simple Connection Provider>>
+ | The connection types to provide to this configuration | | x
+| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before Mule considers it eligible for expiration. This does not mean that the instance expires at the exact moment that it becomes eligible. Mule purges the instances as appropriate. |  | 
+| Encoding a| String |  Canonical name of the encoding. See Oracle's supported encodings at https://docs.oracle.com/javase/7/docs/technotes/guides/intl/encoding.doc.html. |  | 
+|===
+
+
+==== Connection Types
+[[inbound-config_simple-connection-provider]]
+===== Simple Connection Provider
+
+
+Use these parameters to configure the connection provider that connects to the SAP system using a username and password.
+
+
+====== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| SAP system number a| String |  The two-digit system number (sysnr) of the SAP system |  | x
+| SAP client ID a| String |  Three-digit client number. Leading zeros are preserved if they appear in the number. |  | x
+| Destination Name a| String |  Identifier of the destination generated by this connector. If none is set, then a random value is created. |  | 
+| Login language a| String |  The language code used by the connector. The language determines the code page used for communicating between the SAP Adapter and the application server. |  `EN` | 
+| Extended Properties a| Object | Use to pass in all the database server-specific connection parameters |  | 
+| SAP username a| String |  The username the connector uses to log in to SAP |  | x
+| SAP password a| String |  The password associated with the login username |  | x
+| Application server host a| String |  The host of the SAP application server |  | 
+| Message server configuration a| <<MessageServer>> |  The configuration for message server usage |  | 
+| Enable RFC tracing on server a| Boolean | Enable or disable RFC trace on the server |  `false` | 
+| Enable CPIC tracing on server a| Enumeration, one of:
+
+** INHERIT
+** NO_TRACING
+** LEVEL_1
+** LEVEL_2
+** LEVEL_3 |  Enable or disable RFC trace on the server |  | 
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
```

Comment:
```suggestion
| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, and the test exhausts the associated reconnection strategy without passing, then deployment fails. |  | 
```

Reply:
I think this preserves the meaning but I'm not sure. What is the thing that's set to true? Reconnection a?